Sources/Build/BuildPlan.swift

```diff
@@ -192,6 +201,8 @@ public enum TargetBuildDescription {
             return target.bundlePath
         case .clang(let target):
             return target.bundlePath
+        case .mixed(let target):
+            return target.swiftTargetBuildDescription.bundlePath
```

Comment:
I would think that there should be one `target.bundlePath` that matches for both clang and swift.

Reply:
There should be but it didn't matter for prototyping to this point since the `target.swiftTargetBuildDescription == target.clangTargetBuildDescription`.